python/ray/ml/predictors/integrations/torch/torch_predictor.py

```diff
@@ -155,16 +119,7 @@ def predict(
         Returns:
             DataBatchType: Prediction result.
         """
-        self.model.eval()
-
-        if self.preprocessor:
-            data = self.preprocessor.transform_batch(data)
-
-        if isinstance(data, np.ndarray):
-            tensor = torch.tensor(data, dtype=dtype)
-        else:
-            tensor = self._convert_to_tensor(
-                data, feature_columns=feature_columns, dtypes=dtype, unsqueeze=unsqueeze
-            )
 
-        return self._predict(tensor)
+        # TODO: Figure out a better API for this. Overriding predictor just to specify a
+        #  kwarg seems very unecessary.
+        return super().predict(data, dtype=dtype)
```

Comment:
This developer API is also not great. I have to override `predict` just to add a new arg.